### PR TITLE
Update to JavaFX 13

### PR DESCRIPTION
**Pull request by abhinayagarwal**
Update samples and documents to JavaFX 13.

I have also tried to remove data which needs to be updated every time. For example, we explicitly mention "JDK 12" in our documents to be used with JavaFX 12. This has been replaced by "JDK 11 or later".

### CODE REVIEW

File: IDE/Eclipse/Modular/Gradle/README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 
 Version Eclipse: 2019-03 (4.11.0)
 
-Install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
+Download [JDK 11 or later](http://jdk.java.net/) for your operating system.
+Make sure `JAVA_HOME` is properly set to the JDK installation directory.
 
-Download an appropriate [JDK 12](https://jdk.java.net/12/) for your operating system. Make sure `JAVA_HOME` 
-is properly set to the Java 12 installation directory. 
+**N.B**: If you use JDK 12, install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
```
Comment (jperedadnr) [inline IDE/Eclipse/Modular/Gradle/README.md]: **Note** ?

File: IDE/Eclipse/Modular/Java/README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 Version Eclipse: 2019-03 (4.11.0)
 
-Install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
+Download [JDK 11 or later](http://jdk.java.net/) for your operating system.
+Make sure `JAVA_HOME` is properly set to the JDK installation directory.
 
-Download an appropriate [JDK 12](https://jdk.java.net/12/) for your operating system. Make sure `JAVA_HOME` 
-is properly set to the Java 12 installation directory. 
+**N.B**: If you use JDK 12, install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
```
Comment (jperedadnr) [inline IDE/Eclipse/Modular/Java/README.md]: **Note**

File: IDE/Eclipse/Modular/Maven/README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
 Version Eclipse: 2019-03 (4.11.0)
 
-Install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
+Download [JDK 11 or later](http://jdk.java.net/) for your operating system.
+Make sure `JAVA_HOME` is properly set to the JDK installation directory.
 
-Download an appropriate [JDK 12](https://jdk.java.net/12/) for your operating system. Make sure `JAVA_HOME` 
-is properly set to the Java 12 installation directory. 
+**N.B**: If you use JDK 12, install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
```
Comment (jperedadnr) [inline IDE/Eclipse/Modular/Maven/README.md]: **Note**

File: IDE/Eclipse/Non-Modular/Gradle/README.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 
 Version Eclipse: 2019-03 (4.11.0)
 
-Install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
+Download [JDK 11 or later](http://jdk.java.net/) for your operating system.
+Make sure `JAVA_HOME` is properly set to the JDK installation directory.
 
-Download an appropriate [JDK 12](https://jdk.java.net/12/) for your operating system. Make sure `JAVA_HOME` 
-is properly set to the Java 12 installation directory. 
+**N.B**: If you use JDK 12, install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
```
Comment (jperedadnr) [inline IDE/Eclipse/Non-Modular/Gradle/README.md]: **Note**

File: IDE/Eclipse/Non-Modular/Java/README.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 
 Version Eclipse: 2019-03 (4.11.0)
 
-Install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
+Download [JDK 11 or later](http://jdk.java.net/) for your operating system.
+Make sure `JAVA_HOME` is properly set to the JDK installation directory.
 
-Download an appropriate [JDK 12](https://jdk.java.net/12/) for your operating system. Make sure `JAVA_HOME` 
-is properly set to the Java 12 installation directory. 
+**N.B**: If you use JDK 12, install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
```
Comment (jperedadnr) [inline IDE/Eclipse/Non-Modular/Java/README.md]: **Note**

File: IDE/Eclipse/Non-Modular/Maven/README.md [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
 
 Version Eclipse: 2019-03 (4.11.0)
 
-Install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
+Download [JDK 11 or later](http://jdk.java.net/) for your operating system.
+Make sure `JAVA_HOME` is properly set to the JDK installation directory.
 
-Download an appropriate [JDK 12](https://jdk.java.net/12/) for your operating system. Make sure `JAVA_HOME` 
-is properly set to the Java 12 installation directory. 
+**N.B**: If you use JDK 12, install this patch from MarketPlace: `Java 12 support for Eclipse 2019-03 (4.11)`.
```
Comment (jperedadnr) [inline IDE/Eclipse/Non-Modular/Maven/README.md]: **Note**

Comment (moejeb2022) [reply IDE/Eclipse/Non-Modular/Maven/README.md:10]: Thanks a lot ©`°?

**Pull request merged**